TetrisDotnet/Code/Controls/InputManager.cs

```diff
@@ -0,0 +1,47 @@
+using SFML.Window;
+using EventType = TetrisDotnet.Code.Events.EventType;
+
+namespace TetrisDotnet.Code.Controls
+{
+	public class InputManager
+	{
+		public void OnKeyPressed(object sender, KeyEventArgs e)
+		{
+			switch (e.Code)
+			{
+				case Keyboard.Key.Up:
+				case Keyboard.Key.W:
+					Main.eventSystem.ProcessEvent(EventType.InputRotateClockwise);
```

Comment:
Yeah, I agree, but since it can be called from absolutely anywhere I think it's easier for us to keep track this way than to pass it in absolutely everywhere. Although I guess we could do it on a per-case basis of passing it in via constructors.

Worth changing?

Reply:
yes